invokeai/configs/INITIAL_MODELS.yaml

```diff
@@ -80,7 +80,8 @@ trinart-2.0:
      repo_id: stabilityai/sd-vae-ft-mse
    recommended: False
 waifu-diffusion-1.4:
-   description: An SD-1.5 model trained on 680k anime/manga-style images (2.13 GB)
+   description: An SD-2.1 model trained on 5.4M anime/manga-style images (4.27 GB)
+   revision: main
```

Comment:
I'm not familiar with InvokeAI code, so:

- Does this newly introduced `revision` key allows specifying particular commit too?

It would be a desirable thing to support in general, because at some point InvokeAI may want to specify known good commit for predictable download and usage for each model than just blindly following the latest ones available.

If it's not supported yet, I suggest considering adding it.

If you do not want to ever support it, then `revision` would be a bit of misnomer here and IMO `branch` would be more suitable in such case.

Reply:
Agreed that it would be good to support any ref (branch/tag/commit). So we should keep the `revision` name to avoid changing it in the future whenever we do support that.